src/db/events.ts

```diff
@@ -7,6 +7,7 @@ export const events = pgTable('events', {
   id: uuid('id').primaryKey().defaultRandom(),
   name: varchar('name').notNull(),
   description: varchar('description'),
+  eventLink: varchar('event_link'),
```

Comment:
we should rename this just to link, i think we should stop prepending attributes with table name. Not really consistent throughout the tables which can be confusing. For example we do not say description is eventDescription. This is something that slipped through the cracks and we have an issue with this throughout the schema

Reply:
i think the start of this was forumQuestions sadly